### PR TITLE
documentation: ZENKO-1739_Fix_Upgrade_Section

### PR DESCRIPTION
**Pull request by wabernat**
This tidies up several minor defects in the Upgrade section.

- Fixed hierarchization problem.
- Fixed bad note and code block invocations
- Fixed step numbering issue.
- Copyedited.

fixes #ZENKO-1739

### CODE REVIEW

**Comment by wabernat** [comment]
@bert-e approve

File: docs/docsource/Installation/upgrade/upgrading_zenko.rst [hunk 1]
```diff
@@ -16,87 +16,90 @@ To upgrade Zenko:
...
+   .. note::
 
-     Once the upgrade is completed the maintenance flag is not needed
-     and should not be specified for further 1.1.x upgrades.
+      Once this upgrade is complete, the maintenance flag is no
+      longer required for further 1.1.x upgrades.
 
-Upgrade conisderations
-======================
+Upgrade Considerations
```
Comment (anurag4DSB) [inline docs/docsource/Installation/upgrade/upgrading_zenko.rst]: Shouldn't upgrade considerations and dry-run come before the actual upgrade command.
If the user is following the manual step by step, then they might miss this part.

Comment (anurag4DSB) [inline docs/docsource/Installation/upgrade/upgrading_zenko.rst]: Shouldn't upgrade considerations and dry-run come before the actual upgrade command.
If the user is following the manual step by step, then they might miss this part.

Comment (anurag4DSB) [reply docs/docsource/Installation/upgrade/upgrading_zenko.rst]: Something like "Before you upgrade"

**Review by anurag4DSB — CHANGES_REQUESTED** [review]

File: docs/docsource/Installation/upgrade/upgrading_zenko.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/Installation/upgrade/upgrading_zenko.rst]: Good call. Sometimes I don't see the forest for the trees.

**Review by anurag4DSB — APPROVED** [review]

**Review by ssalaues — APPROVED** [review]

**Comment by ssalaues** [comment]
@bert-e bypass_build_status

**Pull request merged**